site/content/en/docs/commands/stop.md

```diff
@@ -20,9 +20,10 @@ minikube stop [flags]
 ### Options
 
 ```
-      --all                   Set flag to stop all profiles (clusters)
-      --keep-context-active   keep the kube-context active after cluster is stopped. Defaults to false.
-  -o, --output string         Format to print stdout in. Options include: [text,json] (default "text")
+      --all                     Set flag to stop all profiles (clusters)
+      --cancel-scheduled-stop   cancel any existing scheduled stop requests
```

Comment:
this seems like a very long name, and since this is already a stop command maybe it is redundent to say stop two times?

Reply:
shortened to cancel-scheduled